packages/components/src/elevation/component.tsx

```diff
@@ -0,0 +1,69 @@
+/**
+ * External dependencies
+ */
+// eslint-disable-next-line no-restricted-imports
+import type { Ref } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import {
+	useContextSystem,
+	contextConnect,
+	PolymorphicComponentProps,
+} from '../ui/context';
+import type { Props } from './types';
+import { ElevationView, ElevationViewProps } from './styles';
+
+const DEFAULT_PROPS: ElevationViewProps = {
+	isInteractive: false,
+	offset: 0,
+	value: 0,
+	active: null,
+	focus: null,
+	hover: null,
+	borderRadius: 'inherit',
+};
```

Comment:
Noting that this is the approach that was agreed in order to set the default values for props only once per component (these default values would otherwise be set also in when computing the styles) in a clear and explicit way.

The styles, in fact, assume that all props are always defined (and does so through the `Required` TypeScript utility)